ComparisonWithOrleans.md

```diff
@@ -0,0 +1,203 @@
+# Orleans and Akka Actors: A Comparison
+
+## Step-By-Step Review
+
+This overview walks along the Technical Report [MSR-TR-2014-41](http://research.microsoft.com/apps/pubs/default.aspx?id=210931) by *Philip A. Bernstein, Sergey Bykov, Allan Geller, Cabriel Kliot, Jorgen Thelin (Microsoft Research)*, for another comparison focusing on Erlang and Riak see [Christopher Meiklejohn’s excellent article](http://christophermeiklejohn.com/orleans.html).
+
+### Introduction
+
+Different focus:
+
+  * Akka as a toolkit for building distributed systems, offering the full power but also exposing the inherent essential complexity.
+  
+  * Orleans restricts applicability in order to allow seamless use without understanding distributed computing.
+  
+#### Lifecycle
+
+  * Orleans Grains do not have a lifecycle, can neither be started nor stopped. Consequently they also cannot fail and be restarted, therefore Orleans does not offer tools for software failure handling—the failure handling aspects concentrate on recovering from hardware crashes.
```

Comment:
I’m not comfortable with this formulation as it conflates two layers: aren’t you talking about the lifecycle of an _Activation_? The paper is very explicit about the fact that a Grain does not have a lifecycle, but _Activations_ are indeed started and stopped (on demand, not explicitly) and there can even be multiple Activations alive concurrently for the same Grain. I’d prefer to be really precise here, since this is an important point.